docs/source/alfacase/03_alfacase_example.rst

```diff
@@ -0,0 +1,121 @@
+.. _alfacase-example:
+
+ALFAcase Example
+================
+
+In this section, we will create a project from scratch with a simple network and insert it on the ALFAsim application.
+
+
+The root level of our :program:`ALFAcase` file is always a :class:`CaseDescription` which has the following attributes:
+For the full reference check the :ref:`alfacase-reference-section`:
+
+.. code-block:: yaml
+
+    name: string    # optional
+    physics: physics_description_schema⠀
+    time_options: time_options_description_schema⠀
+    numerical_options: numerical_options_description_schema⠀
+    ipr_models: ipr_models_description_schema⠀
+    pvt_models: pvt_models_description_schema⠀
+    tracers: tracers_description_schema⠀
+    outputs: case_output_description_schema⠀
+
+    ....
+
+We are going to create a simple case with a PVT model, two nodes and one pipe displaying their output.
+Each section can be inspected over the :ref:`alfacase-reference-section`:
+
+.. code-block:: yaml
+
+    name: basic_case
+    physics:
+    correlations_package: correlation_package_alfasim
+    hydrodynamic_model: hydrodynamic_model_4_fields
```

Comment:
Shouldn't these be indented too? And also the `pvt_models` section below?

Reply:
> > While at it, it would be great to search the docs for "equipments" and fixing those
> 
> @williamjamir did you try this? There was only one incorrect word?

Yes, only one occurrence =)

Reply:
Commit b3e9df5 improves the example section and fix the indentation errors =)